apps/core/views/viewsets/comment.py

```diff
@@ -58,16 +59,22 @@ def create(self, request, *args, **kwargs):
 
     def perform_create(self, serializer):
         parent_article_id = self.request.data.get('parent_article')
-        parent_article = parent_article_id and Article.objects.filter(id=parent_article_id).first()
-        parent_article_is_anonymous = (parent_article and parent_article.is_anonymous) or False
+        parent_article = parent_article_id and Article.objects.get(id=parent_article_id)
+        parent_article_name_type = parent_article and parent_article.name_type != BoardNameType.REGULAR
 
         parent_comment_id = self.request.data.get('parent_comment')
-        parent_comment = parent_comment_id and Comment.objects.filter(id=parent_comment_id).first()
-        parent_comment_is_anonymous = (parent_comment and parent_comment.is_anonymous) or False
+        parent_comment = parent_comment_id and Comment.objects.get(id=parent_comment_id)
+        parent_comment_name_type = parent_comment and parent_comment.name_type != BoardNameType.REGULAR
```

Comment:
63번째 줄과 같은 이유로 `!= BoardNameType.REGULAR`를 제거해야 합니다.

```suggestion
        parent_comment_name_type = parent_comment and parent_comment.name_type
```